cdhweb/projects/exodus.py

```diff
@@ -4,15 +4,15 @@
 from mezzanine.core.models import CONTENT_STATUS_PUBLISHED
 
 from cdhweb.pages.exodus import convert_slug, exodize_attachments, get_wagtail_image, to_streamfield
-from cdhweb.projects.models import OldProject, Project, ProjectsLinkPage
+from cdhweb.projects.models import OldProject, Project, ProjectsLandingPage
 
 
 def project_exodus():
     """exodize all project models"""
     # get the top-level projects link page
     try:
-        project_link = ProjectsLinkPage.objects.get()
-    except ProjectsLinkPage.DoesNotExist:
+        project_link = ProjectsLandingPage.objects.get()
```

Comment:
could update comment and variable name for consistency, but not that important

Reply:
oh, thanks — I did try to update the names, I missed this one